app/models/dog_searcher.rb

```diff
@@ -71,7 +71,9 @@ def self.search(params: {}, manager: false)
   private
 
   def tracking_id_search?
```

Comment:
well what are we really trying to test here?  the whole string is a number?  The problem is that `.to_i` will parse for any number and return true now.

```
[1] pry(main)> "4 seasons".to_i
=> 4
```

maybe the real fix is to make sure the whole string is numbers

```
return @params[:search].to_s.match(/^[0-9]+$/)
```

http://rubular.com/r/Z89CWp7QdZ